corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,163 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
+    domain = models.CharField(max_length=127, db_index=True)
+    connection_settings = models.ForeignKey(
+        ConnectionSettings,
+        on_delete=models.PROTECT,
+    )
+    fhir_version = models.CharField(
+        max_length=12,
+        choices=FHIR_VERSIONS,
+        default=FHIR_VERSION_4_0_1,
+    )
+    frequency = models.CharField(
+        max_length=12,
+        choices=IMPORT_FREQUENCY_CHOICES,
+        default=IMPORT_FREQUENCY_DAILY,
+    )
+    # ID of user or location that will own imported cases
+    owner_id = models.CharField(max_length=32, null=False, blank=False)
```

Comment:
```suggestion
    owner_id = models.CharField(max_length=36, null=False, blank=False)
```
I know we typically do `uuid4().hex`, but technically these IDs are supposed to have dashes bringing them to 36 total digits, like from `str(uuid4())`.  Not sure whether those are encountered in HQ in practice, but it seems prudent to allow for it unless we widely cap UUID fields at 32 chars anyways (do we?).

Reply:
Oh! You are absolutely right. Not sure why I went with 32.